### ISSUE TITLE
Bleadperl v5.17.10-30-g0f539b1 breaks CPAN

### ISSUE THREAD

**Issue by andk** [doherty/App-perlhl#2]
As per subject. Canonical ticket for the issue is
https://rt.perl.org/rt3//Public/Bug/Display.html?id=117427

With the above cited commit, the tests

t/01-ansi.t and t/02-html.t

start to fail. Sample fail report:

http://www.cpantesters.org/cpan/report/0aea7052-98c1-11e2-940d-f673c51cef80

HTH && Regards,

**Comment by tobyink** [comment]
Unfortunately the fix breaks in Perl versions prior to 5.17!

This might do the trick...

```
no if ($] >= 5.017010), warnings => "experimental::smartmatch";
```

**Comment by doherty** [comment]
This is what I get for fixing bugs at midnight `>.<` -- thanks!
